msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -190,6 +190,39 @@ async def on_members_added_activity(
 
 Messages sent between users and bots include internal channel data within the message. This data allows the bot to communicate properly on that channel. The Bot Builder SDK allows you to modify the message structure.
 
+## Send suggested actions
+
+Suggested actions enable your bot to present buttons that the user can tap to provide input. Suggested actions enhance user experience by enabling the user to answer a question or make a selection with a simple tap of a button, rather than typing a response with a keyboard. 
+The buttons remain visible and accessible to the user in the rich cards even after user makes a selection whereas for suggested actions, buttons aren't available. This prevents the user from tapping stale buttons within a conversation and simplifies bot development.
+
+To add suggested actions to a message, set the `suggestedActions` property of the [Activity](/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference) object to specify the list of [CardAction](/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference) objects that represent the buttons to be presented to the user. For more information, see [`SugestedActions`](/dotnet/api/microsoft.bot.builder.messagefactory.suggestedactions)
+
+The following is an example for implementation and experience of suggested actions:
+
+``` json
+"suggestedActions": {
+    "actions": [
+      {
+        "type": "imBack",
+        "title": "Action 1",
```

Comment:
Could we change this to "Equipment request" so that it matches the sample image above?

Reply:
Since this is a JSON payload, we cant change it according to example screenshots. Hence not making the change